lib/cloud/provisioning/awsactions/create_role.go

```diff
@@ -77,6 +77,7 @@ func CreateRole(
 	trustPolicy *awslib.PolicyDocument,
 	tags tags.AWSTags,
 ) (*provisioning.Action, error) {
+
```

Comment:
nit: I have been advised previously to not to have an empty line as a first thing inside Go functions/mehtods.

Reply:
b0547a58a19ddf85a2135a9097b3c369b08cd494